gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -220,13 +223,13 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
         client.
         """
         # Sanity check: Only create a new client if we do not already have one.
-        if 'operations_client' not in self.__dict__:
-            self.__dict__['operations_client'] = operations_v1.OperationsClient(
+        if self._operations_client is None:
```

Comment:
won't `if not self._operation_client` work? I thought it was more idiomatic for python.

Reply:
It's a little bit complicated. Some values can be falsy even if they are non-null, e.g. empty lists and strings.
In this particular case, it's slightly more idiomatic to do `if self._operations_client`, but I personally am fine with explicit checks in cases like this.